addons/l10n_uy/data/account_tax_data.xml

```diff
@@ -1,31 +1,97 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+
     <!-- Account Tax Tags -->
-    <record id="tag_ventas_22" model="account.account.tag">
-        <field name="name">IVA Ventas (22%)</field>
+    <record id="tag_uy_01" model="account.account.tag">
+        <field name="name">Base Imponible</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tag_uy_02" model="account.account.tag">
+        <field name="name">Base Imponible Compras</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tag_uy_03" model="account.account.tag">
```

Comment:
You should name your account tags so that they clearly reference their corresponding report line. As it is here, it will become a complete mess if we add another report at some point.

(you can see how it was done for Switzerland here https://github.com/odoo/odoo/blob/master/addons/l10n_ch/data/account_vat2011_data.xml) 

So, you should change the name and ids of all your tags, here is an example with tag_uy_03, corresponding to line financial_report_line_01_01_01_uy:
- id="vat_tag_01_01_01"
- name="Uruguay VAT Form: Base Compras 22%" 

Please fix all other tags this way